utils/obsproc/NetCDFToIodaConverter.h

```diff
@@ -55,7 +56,43 @@ namespace gdasapp {
       floatMetadataName(fmnames),
       intMetadata(location, imnames.size()),
       intMetadataName(imnames)
-    {}
+    {
+      oops::Log::trace() << "IodaVars::IodaVars created." << std::endl;
+    }
+
+    // Append an other instance of IodaVars
+    void append(const IodaVars& other) {
+      // Check if the two instances can be concatenated
+      ASSERT(nVars == other.nVars);
+      ASSERT(nfMetadata == other.nfMetadata);
+      ASSERT(niMetadata == other.niMetadata);
+      ASSERT(floatMetadataName == floatMetadataName);
+      ASSERT(intMetadataName == intMetadataName);
```

Comment:
What's going on here? It looks like you're making sure the two files are identical - is it just checking for identical structures?

Reply:
@AndrewEichmann-NOAA I'm making sure the objects can be merged. The files can be different but the number and types of metadata need to be the same.